CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ A guide to contributing to opentoken.
     * Following steps may not be needed if forked properly, but in case you can't get to the upstream server.
         1. Configure the upstream server by running `git remote add upstream <repo address>` or editing your `.git/config` under `[remote "upstream"]`, changing `url`.
 3. Create a branch off master `git checkout -b <branch name>`. You don't want to do any work off master as this is where you will pull in upstream changes and merge them into your branches. Also is a good idea to push this remotely `git push -u origin <branch name>`.
-4. Write your code as normal.
+4. Write your code as normal.  Be sure to follow the [Tests-Always-Included Style Guide](https://tests-always-included.github.io/style-guide/)
```

Comment:
This has two spaces after the first sentence. The other sentences only have one.